Microsoft.Kiota.Http.HttpClientLibrary.Tests/Extensions/HttpRequestMessageExtensionsTests.cs

```diff
@@ -101,8 +101,8 @@ public async Task CloneAsyncWithRequestOption()
             Assert.NotNull(clonedRequest);
             Assert.Equal(originalRequest.Method, clonedRequest.Method);
             Assert.Equal(originalRequest.RequestUri, clonedRequest.RequestUri);
-            Assert.NotEmpty(clonedRequest.Options);
-            Assert.Equal(redirectHandlerOption, clonedRequest.Options.First().Value);
+            Assert.NotEmpty(clonedRequest.Properties);

```

Comment:
QQ about that change, we're getting obsolete warnings, is options a property that's only available for dotnet core?

Reply:
This is true. However they both end up pointing to the same thing. 
https://source.dot.net/#System.Net.Http/System/Net/Http/HttpRequestMessage.cs,111

Reply:
we should probably suppress the obsolete warnings for sanity

Reply:
Resolved in latest update